gapis/resolve/memory.go

```diff
@@ -65,7 +65,10 @@ func Memory(ctx context.Context, p *path.Memory) (*service.Memory, error) {
 	list.Atoms[atomIdx].Mutate(ctx, s, nil /* no builder, just mutate */)
 
 	slice := pool.Slice(r)
-	observed := slice.ValidRanges()
+
+	if !p.ExcludeOtherObservations {
+		observed = slice.ValidRanges()
```

Comment:
Was this really huge? I'm surprised it was large enough to require an exclude flag.